dev/com.ibm.ws.microprofile.rest.client_fat_tck/publish/servers/FATServer/jvm.options.debug

```diff
@@ -0,0 +1,4 @@
+-Dwas.debug.mode=true
```

Comment:
Did you want this "was.debug.mode" by default for every FAT run? Fine if so, just wondering.